tests/contrib/redis/test_redis.py

```diff
@@ -17,6 +16,12 @@
 from ..config import REDIS_CONFIG
 
 
+try:
+    from unittest import mock
+except ImportError:
+    import mock  # type: ignore  # noqa
```

Comment:
Why does this avoid the crash during the wheel build step?

Reply:
I put an explanation in the description. Mock is not a ddtrace dependency but it is installed in all dd-trace-py riot venvs. 

So the package always exists in test environments but not in ddtrace wheels.